addons/udes_stock/wizard/change_quant_location.py

```diff
@@ -43,6 +88,24 @@ def create_picking(self):
         if self.reference:
             params['origin'] = self.reference
 
+        # If we allow users to create these pickings for reserved quants, delete
+        # moves/movelines on other pickings.
+        # If we don't allow it the validation in create_picking will error
+        if self._check_reserved_packages_permitted():
+            # Remove moves/move lines for quants in other pickings
+            movelines = packages.get_move_lines()
+            # First split out the movelines from the original pickings
+            # to preserve the original picking and any moves that are for quants
+            # in different pacakges.
+            new_moves = Move.browse()
```

Comment:
Can we give the `new_moves` variable a more descriptive name? This reads as though we're cancelling the new moves created for the new picking.